### ISSUE TITLE
"Updating contact information" guidance page: the content uses some descriptions that are not inclusive 

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#1445]
## Description of issue
Some of the content on this page uses descriptions that benefit sighted users only. 

Describe what is failing
- guidance states "In the box with the percentage failed, there’s a link to review reports"  and "in the box" might not be accessible to users navigating with a screen readers
- the word "view" is used at one place in the page, which is not inclusive.

## Potential fix

- Revise the content to avoid using location cues 
- Replace the "view" with a word inclusive of users who navigate with a screen reader

## Resources

-  UX Content Collective Course "Accessibility for UX Writers and Designers" links to this Intuit page : https://contentdesign.intuit.com/product-and-ui/writing-about-ui/

**Comment by YedidaZalik** [comment]
Made some content fixes using the suggest function
Need to discuss the new content with Phil and also consider if users would benefit from additional screenshots showing the navigation steps

[Ally sprint - Updating contact information](https://docs.google.com/document/d/1CJK76UtgCsRRU2rwmXTlVfNJwwLlhyzdMzHgQV0Rmxk/edit#heading=h.m74m88num1k8)

**Comment by yaelberger-commits** [comment]
Yedida will also review with Phil

**Comment by YedidaZalik** [comment]
Met with Phil to discuss my questions.
Then worked on sections some more based on Phil's suggestions.
Have highlighted the 2 revised sections to show Phil and MSB next week.

**Comment by YedidaZalik** [comment]
Meeting scheduled

**Comment by foudamo** [comment]
Handed over to M-S

**Comment by YedidaZalik** [comment]
Marie Sophie expects to have French ready today

**Comment by YedidaZalik** [comment]
French is ready 
EN and FR in this doc https://docs.google.com/document/d/1CJK76UtgCsRRU2rwmXTlVfNJwwLlhyzdMzHgQV0Rmxk/edit#heading=h.m74m88num1k8
Left to do: Yedida to update pages on GC Articles

**Comment by YedidaZalik** [comment]
Updated EN and FR in GC Articles and cleared cache.
I checked both pages and they look good.
